csaf/src/report/mod.rs

```diff
@@ -0,0 +1,50 @@
+pub mod render;
+
+use crate::discover::DiscoveredAdvisory;
+use std::borrow::Cow;
+use std::collections::{BTreeMap, HashSet};
+use std::path::PathBuf;
+use url::Url;
+use walker_common::utils::url::Urlify;
+
+#[derive(clap::Args, Debug)]
```

Comment:
If we split this up into a clap and plain options variant, please have the `RenderOptions` use the `#[non_exhaustive]` attribute, and use a "builder pattern", like the other options struct now also do.

This allows us to add new fields later, without breaking the API. And I am pretty sure the render options will be changed quite often.